tests/appsec/waf/test_exclusions.py

```diff
@@ -0,0 +1,35 @@
+from utils import scenario, released, weblog, interfaces
+
+
+@released(java="1.3.0", dotnet="?", golang="?", nodejs="?", php_appsec="?", python="?", ruby="?")
```

Comment:
```suggestion
@released(java="1.6.0", dotnet="?", golang="?", nodejs="?", php_appsec="?", python="?", ruby="?")
```

I guess that needs to be v1.6.0.